lib/pb.ml

```diff
@@ -244,25 +246,23 @@ let read_unknown : Wire_type.t -> string Angstrom.t =
   | Thirty_two ->
     take 4
 
-let read_contents : (key, string list) Hashtbl.t Angstrom.t =
-  let add_one : (key, string list) Hashtbl.t -> key -> string -> unit =
+let read_contents : string list KeyMap.t Angstrom.t =
+  let add_one : string list KeyMap.t -> key -> string -> string list KeyMap.t =
     fun h k v ->
-      match Hashtbl.find h k with
-      | exception Not_found -> Hashtbl.add h k [v]
-      | vs -> Hashtbl.add h k (v :: vs)
+      match KeyMap.find_opt k h with
```

Comment:
Could you switch this (and the other occurrences of `find_opt`) to `KeyMap.find` to keep things working on OCaml 4.03 and 4.04?